Foundation/Thread.swift

```diff
@@ -189,6 +217,11 @@ open class Thread : NSObject {
         } else {
             _thread = nil
         }
+#elseif os(Windows)
+        print("stack size request for reservation of size \(stackSize) ignored ...")
```

Comment:
I don't know how to actually do that, it definitely is a TODO.  I think that I do want it to be a log message for now.

Reply:
`debugPrint`, at least?

Reply:
Ah, but thats the real problem - it *is* possible.  I just don't know how we want to plumb that into CF ... since there is only the `pthread_attr` type currently in `_CFCreateThread` which doesn't work for Windows.

Reply:
Personally I am in the camp of avoid logging unless it is truly something actionable. There isnt anything that can be done about it shy of fixing the code. Instead can we just make the stack size unavailable on windows for the setter?

Reply:
Made the setter throw a `NSUnimplemented`.  This is currently the only missing bit of functionality in `NSThread` on Windows.